AzureStore/Ordering/Ordering.Persistent/Migrations/OrderingContextModelSnapshot.cs

```diff
@@ -28,6 +28,9 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.Property<int>("State")
                         .HasColumnType("int");
 
+                    b.Property<double>("Total")
+                        .HasColumnType("float");
```

Comment:
maybe better use `DECIMAL(size, d)` sql type?

Reply:
well float currently ocupied 7 bytes in db and decimal(size, 2) will occupy 5 bytes, so I don't see much difference here